### PR TITLE
Content changes toward being a full SIG

### PR DESCRIPTION
**Pull request by quaid**
- Created new message written from SIG SRE perspective.
- Consolidated previous background content into an organization-related section.
- Refactored paragraphs to have one sentence per line for easier commenting and editing from here.
- Attempted to clarify the tension between "there sure is lots of OpenShift here" and "any SRE practices" to land on "only Open SRE practices here, please, and we're starting where it's
  logical to start: SRE around Kubernetes and OpenShift will get us going and be very valuable

### CODE REVIEW

**Review by schwesig — COMMENTED** [review]
/lgtm

**Comment by hlipsig** [comment]
LGTM

**Review by schwesig — COMMENTED** [review]
/lgtm
/approve

File: README.md [hunk 1]
```diff
@@ -1,95 +1,109 @@
...
 
-This the Operate First repository for Site Reliability Engineering (SRE)
-documentation. Here we intend to collect documentation on the nitty-gritty
-aspects of building and operating an effective SRE organisation.
+When starting the Operate First project, we looked around and said, "There is no Open group of SRE practitioners.
+Nowhere to discuss and document all the Why, What, and How of our discipline.
+We should fix that."
 
-## Who runs this repository?
+So we are fixing that, and you are invited to participate in creating this [community of practice](https://en.wikipedia.org/wiki/Community_of_practice) around Site Reliability Engineering (SRE).
```
Comment (jeremyeder) [inline README.md:9]: somewhere I thought we were using SIG SRE, so why is it CoP here? https://en.wikipedia.org/wiki/Special_interest_group

File: README.md [hunk 2]
```diff
@@ -1,95 +1,109 @@
...
 
-Hello! We're the Red Hat SIG-SRE group, a cross-team group of people inside Red
-Hat with an interest in promoting and developing strong SRE culture. As a
-result you can expect some of the material here to be a little slanted toward
-the way we do things here inside Red Hat, but as we're doing our best to stick
-to what are generally accepted to be good SRE practices you should be able to
-make use of it with, at most, minor modification.
+This is SIG SRE, the Special Interest Group (SIG) for Site Reliability Engineering (SRE) in the Operate First project.
+
+We focus on a core part of the Operate First goal to improve managed services by fully Opening all aspects of the cloud environment.
```
Comment (jeremyeder) [inline README.md:13]: With a particular focus on observability, managing fleets, incident response, operability and similar core disciplines.

File: README.md (hunk 1 above)
Comment (quaid) [reply README.md:9]: ok, let's make it clear it's a SIG and also a CoP, same thing but the latter is more encompassing

**Pull request merged**